zap2epg.py

```diff
@@ -371,7 +422,20 @@ def printEpisodes(fh):
                                     if edict['epfilter'] is not None and edict['epgenres'] is not None:
                                         genreNewList = genreSort(edict['epfilter'], edict['epgenres'])
                                         for genre in genreNewList:
-                                            fh.write("\t\t<category lang=\"" + lang + "\">" + re.sub('&','&amp;', genre) + "</category>\n")
+                                            fh.write("\t\t<category lang=\"" + lang + "\">" + genre + "</category>\n")
```

Comment:
This either needs `html.escape(genre, quote=True )` or put the `re.sub('&','&amp;', genre)` back in to handle an & in EPG data on or around 12/03/2023 which is causing some parsing issues.